test/langtools/tools/javac/valhalla/lworld-values/UnknownTypeDefault.out

```diff
@@ -0,0 +1,3 @@
+UnknownTypeDefault.java:11:30: compiler.err.cant.resolve.location: kindname.class, Y, null, null, (compiler.misc.location: kindname.class, UnknownTypeDefault, null)
+UnknownTypeDefault.java:12:32: compiler.err.cant.resolve.location: kindname.class, y.Z, null, null, (compiler.misc.location: kindname.class, UnknownTypeDefault, null)
+2 errors
```

Comment:
I submitted it first with tabs instead of spaces, and corrected the spaces without fixing the issue, assuming 4 space tabs.
I should have caught this in the tier1 langtools test pipeline, I was just looking in the wrong place.